subsys/mgmt/mcumgr/lib/cmd/os_mgmt/Kconfig

```diff
@@ -19,9 +19,10 @@ config OS_MGMT_RESET_MS
 	  before performing the reset.  This delay allows time for the mcumgr
 	  response to be delivered.
 
-config OS_MGMT_RESET_HOOK
+config MGMT_MCUMGR_GRP_OS_OS_RESET_HOOK
```

Comment:
Sorry for asking to change this again, but I have agreed that the prefix `MGMT_` is too much, and `MCUMGR_` is distinctive enough.